bin/deploy

```diff
@@ -169,23 +177,52 @@ main() {
   local conda_packages
   conda_packages="conda${pyver_prefix}_${deploy_mode}-${pkg_postfix}.yml"
 
+  if [[ $FIXED_ENVREF != '' ]]; then
+    # (try to) get latest branch SHA1 and assign it to LSST_SPLENV_REF
+    # if the REF is not a branch (or a tag), the reftip will be empty
+    rawreftip=$(git ls-remote https://github.com/lsst/scipipe_conda_env.git ${ENVREF})
+    reftip=$(echo "$rawreftip" | cut -c1-7)
+    reftype=$( echo "$rawreftip" | cut -f 2 -d ' ' | cut -f 2 -d'/' )
+    echo ">> $reftype"
```

Comment:
I think this debug line should be removed or else be some informative message saying something about the default conda env being installed.